### ISSUE TITLE
BPM overlay for desktop Steam steals controller input from games

### ISSUE THREAD

**Issue by LiamDawe** [ValveSoftware/steam-for-linux#6495]
#### Your system information

* Steam client version (build number or date): Sept 17
* Distribution (e.g. Ubuntu): Manjaro KDE
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible (plus steps):

Enable the new Big Picture Overlay for Rocket League on desktop mode with a Steam Controller.

Enjoy having the overlay steal all input making it impossible to play and revert back to the desktop overlay.

Basically, even with the overlay closed, you can hear movements with the stick acting like the overlay is there and taking the input.

**Comment by PSebs** [comment]
I can say I experienced this same issue with multiple games with a steam controller and a dinput controller.

**Comment by Managor** [comment]
I'll chime in with Bioshock Remastered. This bug can be circumvented by opening BPM and then launching the game.

**Comment by julikaiba** [comment]
same for me in Mad Max, Portal and Serial Cleaner. You can hear the inputs, when reopening the overlay, the selection hasnt changed though. Also if you press "a" a lot when having this issue, might sometimes  cause the game to minimize randomly. Also steam doesnt have to be in BPM, only the overlay needs to.

**Comment by loisgomez** [comment]
Affected by this same bug. Can hear the overlay when i interact with the controller.

**Comment by Managor** [comment]
This doesn't happen any more but it's because launching BPM overlay from the desktop client was rolled back. I'm assuming that Plagman is still trying to get it to work behind the scenes so it's still a bug, just not an active one to the user.

**Comment by blastermaster77** [comment]
what is the status of this bug?

**Comment by PSebs** [comment]
The Big Picture Overlay on the desktop no longer seems to be stealing my steam controller inputs as of the recent Dec 16 Steam Beta update

**Comment by Managor** [comment]
I can confirm. Tried Hitman 2 about a week ago and there was no problems while BPM overlay was on.

**Comment by kisak-valve** [comment]
Closing per the last couple comments.
